integration-cli/docker_cli_run_test.go

```diff
@@ -471,18 +472,22 @@ func (s *DockerSuite) TestRunVolumesFromInReadWriteMode(c *check.C) {
 }
 
 func (s *DockerSuite) TestVolumesFromGetsProperMode(c *check.C) {
-	// TODO Windows: This test cannot yet run on a Windows daemon as Windows does
+	// TODO
```

Comment:
Can you keep `TODO Windows:` on a single line together, with the colon - that's how I'm keeping track of them for fixing later. If necessary add a `TODO win2lin` on a seperate line. But I really don't want to lose the `TODO Windows:` nomenclature otherwise `git grep` is next to impossible. In other places too.